lib/filter.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 
-const urlFn = require('url');
+const { parseUrl } = require('url');
```

Comment:
should be `{ parse }`, which is a property of `url`.

Reply:
Fixed and rebased.